scrapi/base/__init__.py

```diff
@@ -171,36 +204,61 @@ def get_properties(self, result, property_list):
 
         properties = {}
         for item in property_list:
-            prop = (result.xpath('//dc:{}/node()'.format(item), namespaces=self.NAMESPACES) or [''])
-            prop += (result.xpath('//ns0:{}/node()'.format(item), namespaces=self.NAMESPACES) or [''])
-
-            properties[item] = prop if len(prop) > 1 else prop[0]
+            prop = (
```

Comment:
Can you remove the surrounding `()` it makes the value look like it should be a tuple when its not.